CONTRIBUTING.md

```diff
@@ -5,10 +5,12 @@ The following is a set of guidelines for contributing to Bonita BPM documentatio
 
 ## Pull requests
 
-Each minor version of Bonita BPM has its own Git branch to manage its related documentation. (Minor version means X.Y, for example 7.3 or 7.4).
-
+Each minor version of Bonita BPM has its own Git branch to manage its related documentation. (Minor version means X.Y, for example 7.3 or 7.4).  
 For instance, the **7.3** branch is where you have to open pull requests when your contribution applies to all 7.3 versions of Bonita BPM.
 
+A branch should be name in kebab case prefixes by the type of the feature the branch has bcreated for (fix, chore, howto,...).  
```

Comment:
chore is maintenance job. 
In fact, I updated the list we have in other repos because all terms may be confusing, IMHO. 
I will add them all then.